docs/source/release/v6.3.0/diffraction.rst

```diff
@@ -5,78 +5,96 @@ Diffraction Changes
-- Fix issue in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` where in some cases you end up with zeros as output.
-- Fix bug such that attenuation calculated in :ref:`AnvredCorrection <algm-AnvredCorrection>` is now accurate to within 0.5% for typical muR.
+- Identification in :ref:`AlignComponents <algm-AlignComponents>` of the first and last ``detector-ID`` for an instrument component with unsorted detector-ID's as the smallest and largest ``detector-ID`` values.
+- Fixed a bug such that attenuation calculated in :ref:`AnvredCorrection <algm-AnvredCorrection>` is now accurate to within 0.5% for typical muR.
+- Restored behavior in :ref:`ConvertUnits <algm-ConvertUnits>` where negative ``TOF`` converts to negative ``d-Spacing`` when ``DIFA==0`` .
+- :ref:`LoadPDFgetNFile <algm-LoadPDFgetNFile>` now returns standard units for atomic distance rather than the label.
 - The integration range has been corrected inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`.
-- Fix problem with the create_vanadium action when running with tt_mode=Custom in the ISIS PEARL powder diffraction scripts. Create a separate Vanadium file for each different custom grouping file rather than one for all custom runs
+- :ref:`SaveFocusedXYE <algm-SaveFocusedXYE>` now correctly writes all spectra to a single file when ``SplitFiles`` is ``False``. Previously it wrote only a single spectrum.
+- For processing vanadium run it no longer finds the environment automatically in :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>`.
+- Fixed an issue in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` where in some cases users ended up with zeros as output.
+- Fixed a problem with the ``create_vanadium`` action when running with ``tt_mode=Custom`` in the ISIS PEARL powder diffraction scripts. Created a separate Vanadium file for each different custom grouping file rather than one for all custom runs
+
+Deprecation
+############
+- ``GetDetOffsetsMultiPeaks``, which is deprecate since v6.2.0, is removed.
+- ``CalibrateRectangularDetectors``, which is deprecate since v6.2.0, is removed.
 
 Engineering Diffraction
 -----------------------
 New features
 ############
-- Now support two texture grouping schemes: Texture20 (10 groups per bank, 20 in total) and Texture30 (15 groups per bank, 30 in total) for ENGIN-X in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the bankID log values saved with focused data, so this means the UI will not load in previously focused .nxs files.
+- Now supports two texture grouping schemes: ``Texture20`` (10 groups per bank, 20 in total) and ``Texture30`` (15 groups per bank, 30 in total) for ``ENGIN-X`` in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the ``bankID`` log values saved with focused data, so this means the UI will not load in previously focused ``.nxs`` files.
```

Comment:
This looks like the most impressive thing on this page! Here's an image from @RichardWaiteSTFC for this note
![TextureGroup20](https://user-images.githubusercontent.com/55980573/148940797-7a87deb2-0e8f-41a0-ab13-834583ab6032.png)
.